### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lkapustka**

### CODE REVIEW

File: src/views/Logout.vue [hunk 1]
```diff
@@ -1,9 +1,15 @@
 <script>
-import { defineComponent, inject } from "@vue/composition-api";
+import { inject } from "@vue/composition-api";
 
-export default defineComponent({
+export default {
```
Comment (daksamit) [inline src/views/Logout.vue:4]: question: why did you remove defineComponent wrapper function?

Comment (lkapustka) [reply src/views/Logout.vue:4]: Because i have read this topic: [export default vs. defineComponent](https://stackoverflow.com/questions/66956931/which-way-better-to-create-vue-component-export-default-vs-definecomponent-vs-n). I don't use TS so i removed it.
